### PR TITLE
APPS-1384 Create a pipeline named after folder when building from repo

### PR DESCRIPTION
**Pull request by commandlinegirl**

### CODE REVIEW

File: src/python/dxpy/nextflow/nextflow_templates.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def is_importer_job():
     return dxapp
 
 
-def get_nextflow_src(inputs=None, profile=None):
+def get_nextflow_src(inputs=None, profile=None, resources_dir=None):
```
Comment (YuxinShi0423) [inline src/python/dxpy/nextflow/nextflow_templates.py:42]: Missing docstring

File: src/python/dxpy/nextflow/nextflow_utils.py [hunk 2]
```diff
@@ -7,8 +7,19 @@
...
-def get_resources_subpath():
-    return "/home/dnanexus/nfp"
+def get_resources_dir_name(resources_dir):
+    """
+    :param resources_dir: Directory with all source files needed to build an applet. Can be an absolute or a relative path.
+    :type resources_dir: str or Path
+    :returns: The name of the folder
+    :rtype: str
+    """
+    if resources_dir == None:
```
Comment (YuxinShi0423) [inline src/python/dxpy/nextflow/nextflow_utils.py:17]: This is not covered previously. Is there a use case for it?

File: src/python/dxpy/nextflow/nextflow_templates.py (hunk 1 above)
Comment (YuxinShi0423) [reply src/python/dxpy/nextflow/nextflow_templates.py:42]: Just saw it. Thanks!

File: src/python/dxpy/nextflow/nextflow_utils.py (hunk 2 above)
Comment (commandlinegirl) [reply src/python/dxpy/nextflow/nextflow_utils.py:17]: This is for when the resources_dir is no specified (and we want '', for example here: https://github.com/dnanexus/dx-toolkit/pull/824/files#diff-7080b6a8931911206ca88463752bac6d538e2e4b772dbcfaafd1fb2e4ae7e6b6R21

Comment (YuxinShi0423) [reply src/python/dxpy/nextflow/nextflow_utils.py:17]: Got it. Nice catch!

**Review by YuxinShi0423 — APPROVED** [review]
Thank you @commandlinegirl  for solving this issue!

**Pull request merged**